mesh/mesh.hpp

```diff
@@ -274,6 +275,9 @@ class Mesh
    /// A list of all unique boundary attributes used by the Mesh.
    Array<int> bdr_attributes;
 
+   /// Named sets of element and boundary element attributes
+   AttributeSets attribute_sets;
```

Comment:
What do people think: should we have public member data or accessor member function? I'm fine either way

Reply:
What is the reason for having any public access to `attribute_sets`? Isn't it used only internally by `Mesh`? Why not make it `protected`?

Reply:
Since `attributes` and `bdr_attributes` are public (right above), this can probably follow the pattern.

Reply:
Yes, @vladotomov is correct. I was just following this pattern (and hoping no one would notice). I'm also fine either way.